src/templates/header.hbs

```diff
@@ -20,152 +20,229 @@
-                  IP Intelligence Data
-                </span>
-              </a>
-            </li>
-            <li>
-              <a class="nav-item" href=''>
-                <span>
-                  Integrate Databases
-                </span>
-              </a>
-            </li>
-            <li>
-              <a class="nav-item" href=''>
-                <span>
-                  Integrate Web Services
-                </span>
-              </a>
-            </li>
-            <li>
-              <a class="nav-item" href='hc/en-us/articles/4408252036123-GeoIP-Corrections'>
-                <span>
-                  GeoIP Corrections
-                </span>
-              </a>
-            </li>
-          </ul>
-          <a class="nav-category nav-item" href="/hc/en-us/categories/1260801446690-Account-and-Purchasing">
-            <span class="nav-icon" role="img">
-              <include src="svg/icon-account-and-purchasing.svg"></include>
-            </span>
-            <span class="nav-label">Account</span>
-          </a>
-          <ul class="sub-nav-items account-and-purchasing">
-            <li>
-              <a class="nav-item" href=''>
-                <span>
-                  Manage my Account
-                </span>
-              </a>
-            </li>
-            <li>
-              <a class="nav-item" href=''>
-                <span>
-                  Secure my Account
-                </span>
-              </a>
-            </li>
-            <li>
-              <a class="nav-item" href=''>
-                <span>
-                  Manage my License Keys
-                </span>
-              </a>
-            </li>
-            <li>
-              <a class="nav-item" href=''>
-                <span>
-                  Purchases, Subscriptions, and Refunds
-                </span>
-              </a>
-            </li>
-          </ul>
-          <a class="nav-category nav-item" href="/hc/en-us/categories/4407628905883-Data-Privacy-and-Licenses">
-            <span class="nav-icon" role="img">
-              <include src="svg/icon-account-and-purchasing.svg"></include>
-            </span>
-            <span class="nav-label">Data Privacy</span>
-          </a>
-          <ul class="sub-nav-items data-privacy-and-licenses">
-            <li>
-              <a class="nav-item" href=''>
-                <span>
-                  Placeholder
-                </span>
-              </a>
-            </li>
-          </ul>
-        </div>
       </nav>
     </div>
   </div>
 </header>
+
+<div class="header-sidebar header-sidebar-hidden">
+  <nav class="sidebar-list">
+    <a class="nav-item contact-us-sidenav" href="{{page_path 'new_request'}}">
+      <span class="nav-icon" role="img">
+        <include src="svg/icon-new-request.svg">
+      </span>
+      <span class="nav-label">Contact Us</span>
+    </a>
+    <div class="nav-category is-closed" data-category-name="minfraud-services">
```

Comment:
This is a very, very thorough implementation of an accessible treeview. Let's implement the html as recommended, and then reevaluate the keyboard support after we have the correct markup in place.


https://www.w3.org/TR/wai-aria-practices-1.1/examples/treeview/treeview-2/treeview-2b.html